templates/cmp/army-number-search.html

```diff
@@ -1,7 +1,12 @@
 {% extends "base.html" %}
 {% block title %}Army Number Search{% endblock %}
 {% block content %}
+
 <div>
+{% load static %}
+<img src="/static/cmp/flag.svg" style="width:25%; height:auto;" alt="The Flag of the British Army. A lion wearing a crown on two crossed swords. ">
```

Comment:
this image should be loaded from {% static 'path/to/image' %} 

`<img id="british_flag" src="{% static 'cmp/flag.svg' %}" alt="The Flag of the British Army. A lion wearing a crown on two crossed swords. ">`   

The id tag will allow us to easily target this img via CSS if needed.